include/zephyr/bluetooth/classic/spp.h

```diff
@@ -0,0 +1,95 @@
+/** @file
+ *  @brief Bluetooth SPP Protocol handling.
+ */
+
+/*
+ * Copyright 2024 Xiaomi Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __BT_SPP_H
+#define __BT_SPP_H
+
+/**
+ * @brief SPP
+ * @defgroup SPP
+ * @ingroup bluetooth
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/bluetooth/buf.h>
+#include <zephyr/bluetooth/conn.h>
+
+/** @brief spp app call back structure. */
+struct bt_spp_cb {
+	void (*connected)(struct bt_conn *conn, uint8_t spp_id);

```

Comment:
How to know the correspondence between spp_id and uuid?

Reply:
let's define new api with 	void (*connected)(struct bt_spp *spp, uint8_t port);